packages/pockethost.io/src/util/stores.ts

```diff
@@ -1,3 +1,22 @@
 import { writable } from 'svelte/store'
 
-const instanceCreationWidgetName = writable(0)
+export const globalUserData = writable({})
+export const isUserLoggedIn = writable(false)
+export const isUserVerified = writable(false)
+
+// Update some derived stores based on the logged-in state of the user
+globalUserData.subscribe((data) => {
+  // Check if the email property exists
+  if (data?.user?.email) {
+    isUserLoggedIn.set(true)
+  } else {
+    isUserLoggedIn.set(false)
+  }
+
+  // Check if the user has verified their email
+  if (data?.user?.verified) {
```

Comment:
alternative:  `isUserVerified.set(!!data?.usere?.verified)`

Reply:
Great way to simplify! I'll update now.

Reply:
@benallfree I found the type in the pocketbase project `UserAuthResponse`, but it's not exported. I'm still learning Typescript, so in situations like this, do we just make a copy of the type on our side? Or should we request Pocketbase to make an update to the SDK to export their types for us to use?

The `User` type is defined, and that's what I'm using now, but `UserAuthResponse` is actually what we need for the store.